src/data/multiset/basic.lean

```diff
@@ -37,6 +37,17 @@ instance has_decidable_eq [decidable_eq α] : decidable_eq (multiset α)
 | s₁ s₂ := quotient.rec_on_subsingleton₂ s₁ s₂ $ λ l₁ l₂,
   decidable_of_iff' _ quotient.eq
 
+private def sizeof_core [has_sizeof α] (s : multiset α) : ℕ :=
```

Comment:
I would either make this protected or provide lemmas that relate `list.sizeof` and `multiset.sizeof`

Reply:
I made it `protected` and renamed it to `sizeof`. I don't think we can relate `multiset.sizeof` to `list.sizeof`, because multiset didn't even come with a `sizeof` implementation. Technically, `multiset` is a function of `Type u -> Type u` as opposed to an inductive which is probably why it didn't have a meaningful `sizeof` in the first place. 

Having `sizeof` sit on it, feels ok in this case. Any objections?